lib/reverse_string.rb

```diff
@@ -1,5 +1,13 @@
 def reverse_characters(str)
+    array = [str.split("")]
```

Comment:
Something to be careful of here is that the `split` method returns an array already. Let's say you split the string `"Hello"` and store it in a variable `x`. So `x` will be **an array** – `["H", "e", "l", "l", "o"]`.

If you then do `array = [x]`, you'll be putting **an array** into an array :) So you would have `[["H", "e", "l", "l", "o"]]` (Notice the extra square brackets). That's probably not what you had intended to create, so it will throw off the rest of your algorithm.